src/test/java/kitchenpos/application/OrderServiceTest.java

```diff
@@ -0,0 +1,217 @@
+package kitchenpos.application;
+
+import static kitchenpos.domain.OrderFixture.*;
+import static kitchenpos.domain.OrderLineItemFixture.*;
+import static kitchenpos.domain.OrderTableFixture.*;
+import static org.assertj.core.api.Assertions.*;
+import static org.mockito.BDDMockito.*;
+
+import java.util.Arrays;
+import java.util.Collections;
+import java.util.List;
+import java.util.Optional;
+
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.EnumSource;
+import org.mockito.InjectMocks;
+import org.mockito.Mock;
+import org.mockito.junit.jupiter.MockitoExtension;
+
+import kitchenpos.dao.MenuDao;
+import kitchenpos.dao.OrderDao;
+import kitchenpos.dao.OrderLineItemDao;
+import kitchenpos.dao.OrderTableDao;
+import kitchenpos.domain.Order;
+import kitchenpos.domain.OrderLineItem;
+import kitchenpos.domain.OrderStatus;
+import kitchenpos.domain.OrderTable;
+
+@DisplayName("주문 서비스 테스트")
+@ExtendWith(MockitoExtension.class)
+class OrderServiceTest {
+    @Mock
+    private MenuDao menuDao;
+    @Mock
+    private OrderDao orderDao;
+    @Mock
+    private OrderLineItemDao orderLineItemDao;
+    @Mock
+    private OrderTableDao orderTableDao;
+
+    @InjectMocks
+    private OrderService orderService;
+
+    @DisplayName("주문 등록 API - 빈 주문 항목")
+    @Test
+    void create_orderLineItems_empty() {
+        // given
+        Order order = orderParam(1L, Collections.emptyList());
+
+        // when, then
+        assertThatThrownBy(() -> orderService.create(order))
+            .isInstanceOf(IllegalArgumentException.class);
+    }
+
+    @DisplayName("주문 등록 API - 동록 되어 있지 않은 주문 항목 메뉴")
```

Comment:
```suggestion
    @DisplayName("주문 등록 API - 등록 되어 있지 않은 주문 항목 메뉴")
```
이건 그냥 보여서 수정해둡니다 ㅎ.ㅎ;;